twitter-clone/src/common/components/Header/Header.tsx

```diff
@@ -21,7 +14,7 @@ const Header = ({ userName }: { userName: string }) => {
   return (
     <header className={styles.header}>
       <div className={styles.userName}>{userName}</div>
-      <Button onClick={logOut} additionalStyles={buttonStyles}>
+      <Button onClick={logOut} styleType="secondary">
```

Comment:
style type can be renamed as 'variant'. this approach is used in Chakra UI for example, can be more informative